### PR TITLE
Updated FAQ Layout

### PR DESCRIPTION
**Pull request by callmehiphop**
* Changed all questions within the FAQs to use h2 & h3 tags since h4 & h5 looks pretty similar to body text.
* Moved get help section into a separate markdown file to allow for easy addition of project specific faqs
* Added troubleshooting section to main README

### CODE REVIEW

File: faq/get-help.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+If you're experiencing a bug with the code, or have an idea for how it can be improved, *please* create a new issue on GitHub so we can talk about it.
+
+  - [`gcloud-golang` on GitHub][gh-golang]
+  - [`gcloud-java` on GitHub][gh-java]
+  - [`gcloud-node` on GitHub][gh-node]
+  - [`gcloud-python` on GitHub][gh-python]
+  - [`gcloud-ruby` on GitHub][gh-ruby]
+
+
+[gcloud-cli]: https://cloud.google.com/sdk/gcloud/
```
Comment (stephenplusplus) [inline faq/get-help.md]: Not used here.

File: faq/readme.md [hunk 2]
```diff
@@ -1,67 +1,12 @@
...
-If you're experiencing a bug with the code, or have an idea for how it can be improved, *please* create a new issue on GitHub so we can talk about it.
-
-  - [`gcloud-golang` on GitHub][gh-golang]
-  - [`gcloud-java` on GitHub][gh-java]
-  - [`gcloud-node` on GitHub][gh-node]
-  - [`gcloud-python` on GitHub][gh-python]
-  - [`gcloud-ruby` on GitHub][gh-ruby]
-
-
-[gcloud-cli]: https://cloud.google.com/sdk/gcloud/
```
Comment (stephenplusplus) [inline faq/readme.md]: Wait, we need this!

**Comment by stephenplusplus** [comment]
Let's move the "getting more help" guide to a new troubleshooting/readme.md. Ideally, this repo should be easily consumable as well.

File: troubleshooting/readme.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+<a name="get-help"></a>
```
Comment (stephenplusplus) [inline troubleshooting/readme.md]: Pop a title in here

Comment (callmehiphop) [inline troubleshooting/readme.md]: Do you want me to add them to faq.md as well? They don't have any either.

Comment (stephenplusplus) [inline troubleshooting/readme.md]: Sorry, a "# Troubleshooting" title.

**Comment by stephenplusplus** [comment]
Thanks!

**Pull request merged**